docs-chef-io/content/inspec/resources/file.md

```diff
@@ -587,3 +596,11 @@ return `true` if your file has a mode with greater permissions than specified.
       it { should_not be_more_permissive_than('0644') }
       it { should be_more_permissive_than('0000') }
     end
+
+### `be_inherit`
```

Comment:
Please rename this to "be_inherited".

Reply:
Thanks, @clintoncwolfe. Did the required changes.